app/views/shared/finalize.html.erb

```diff
@@ -0,0 +1,30 @@
+<div id="maincontent" class="row">
+
+  <% if @registration.expired? %>
+    <strong>Registration for <%= @registration.dsp_email %> has expired.
+            Please contact <%= mail_to ApplicationMailer.default[:from] %>.</strong>
+  <% elsif @already_used %>
+    <strong>This link is no longer valid; it may have already been used.</strong>
+  <% else %>
+    <p>Thank you.</p>
+    <p>Elevated access registration confirmed for <strong><%= @registration.dsp_email %></strong>.</p>
+    <p>A HathiTrust staff member will approve your registration and respond shortly by e-mail.</p>
+    
+    <% if @institution.entityID && @institution.shib_authncontext_class %>
+      <p>Your registration indicates you have access to mult-factor authentication (MFA).</p>
+      <p>You can test your ability to log in with multi-factor authentication using the link
+        <%= raw @institution.mfa_test_link %>.
+    <% elsif @registration.mfa_addendum? %>
+      <% # TODO HT-1212: we want to direct users out and back to pick up MFA info if possible %>
+      <p>Your registration indicates you have access to mult-factor authentication (MFA); however,
```

Comment:
The MFA addendum means the user attested they used MFA to log in to HathiTrust, but that we can't detect that via the auth context class. I don't think we need the language about 'please contact' but some reminder from the MFA addendum form about the expectations for MFA might be helpful.